packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -256,7 +256,6 @@ export default class SubmodeLayout extends Component<Signature> {
         margin-right: 0;
         background-color: var(--boxel-ai-purple);
         box-shadow: var(--boxel-deep-box-shadow);
-        z-index: 100;
```

Comment:
is there a way we can make this more informational about the z index position relative to other elements? 

`z-index: calc(var(--boxel-modal-z-index) - 1);` maybe?

Reply:
That does work.